views/indexPage.js

```diff
@@ -0,0 +1,176 @@
+	        for(var j = 0; j < 30; j++){
+	          result += '<tbody><tr><th scope="row">' + (j + 1) + 
+	          '</th><td>' + data[i].datetime[j] + 
+	          '</td><td>' + data[i].humidity[j] + 
+	          '</td><td>' + data[i].temperature[j] + 
+	          '</td><td>' + data[i].wind[j] + 
+	          '</td></tbody>';
+	        }
+	    }	  	
+	  	return '<table border="1">' + result + '</table>';
+	};
+	var Form = function () {		
+		return '<form method="POST" class="form-inline">' +			   
+			   '<label>Місто:</label>' +
+			   '<input type="text" class="form-control" name="nameForm"/>' +
+			   '<label>Параметр:</label>' +
+			   '<select name="paramWeather" class="form-control">' +
+			   '<option value="0">Вологість</option>' +
+			   '<option value="1">Температура</option>' +
+			   '<option value="2">Вітер</option></select>' +
+			   '<input class="btn btn-default" type="submit" value="Пошук">' +
+			   '</form>';
+	};
+	var FormStat = function () {
+		return '<form method="POST" class="form-inline">' +
+			   '<label>Статистика:</label>' +
+			   '<select name="Stat" class="form-control">' +
+			   '<option value="0">------</option>' +
+			   '<option value="1">Всі точки</option>' +
+			   '<option value="2">Середнє по всіх</option>' +
+			   '</select><input class="btn btn-default" type="submit" value="Показати">' +
+			   '</form>';
+	};
+	var searchByParams = function(params){
+	    var result = ""	  	
+		  , data = getViewData(params)
+		  , averageAllData = mainModule.averageArray(data)
+		  , maximum = mainModule.maximumArray(data);      
+    	
+	    if (!data.length) {
+			return [];
+		}	
+		if (!params) {
+			return "";
+		}	
+	    if(params.paramWeather == 2){        
+	        result = '<hr><h2 class="text-center bg-danger">Максимальне значення вітру по місту ' + 
+	                 params.nameForm + 
+	                 ' - ' + 
+	                 averageAllData[2] + 
+	                 '</h2>';         
+	    } else if (params.paramWeather == 1){
+	        result = '<hr><h2 class="text-center bg-danger">Максимальне значення температури по місту ' + 
+	                 params.nameForm + 
+	                 ' - ' + 
+	                 maximum[1] + 
+	                 '&deg;С</h2>';             
+	    } else if (params.paramWeather == 0){        
+	        result = '<hr><h2 class="text-center bg-danger">Максимальне значення вологості по місту ' + 
+	                 params.nameForm + 
+	                 ' - ' + 
+	                 maximum[0] + 
+	                 '%</h2>';      
+	    }      
+	    return result;
+	};
+	var searchForm = function(params){
+		var result = ""			
+			, data = mainModule.getAll()
+			, averageAllData = mainModule.averageArray(data);
+			if (!data.length) {
+				return [];
+			}	
+			if (!params) {
+				return "";
+			}			
+		    if( params.Stat == 1 ){

```

Comment:
this `if` should not be so big, move some parts into functions, function length should be 30 lines, not more